apps/wing/src/analytics/collectors/cli-collector.ts

```diff
@@ -28,7 +28,7 @@ export class CLICollector extends Collector {
     return {
       platform: platform.map((p: string) => basename(p)).join(","), // only report the platform name, not the full path
       target: determineTargetFromPlatforms(platform),
-      options: `${JSON.stringify(this.cmd.opts())}`,
```

Comment:
So I think this will result in a nested object right? If I recall correctly @ekeren advised against this as there are complications with nested JSON objects in segment. (unless Im misremembering)

Reply:
yes, nested json is not good

Reply:
may I ask why?
I saw that segment handles them pretty well- and also you can use them as an additional filter-
without it we cannot parse the command option via segment (we won't be able to pass the language and the template to the slack notification, but as a json stringified string)

Reply:
segment deals with it well, but amplitude (which is where we actually consume the analytics via graphs) doesn't know how to handle them.

we can concat keys

e.g. 
```
{
  a: { 
    b: 
      {
        c: "v1"
      } 
    },
      d: "v2" 
  }
  d:"v3"
 }
```
==> 
```
{
a_b_c: "v1",
a_d: "v2",
d: "v3"
}
```

Reply:
Other consumers (like mixpanel) and google analytics are also not great with these nested json values...

Reply:
added :) 
![image](https://github.com/winglang/wing/assets/39455181/f9b5f07c-1a0d-4d89-af6d-83b51be066c2)
(The flattening is at our end now)